zettarepl/replication/task/snapshot_owner.py

```diff
@@ -43,7 +43,7 @@ def owns_dataset(self, dataset: str):
 
         raise ValueError(self.side)
 
-    def owns_snapshot(self, parsed_snapshot_name: ParsedSnapshotName):
+    def owns_snapshot(self, dataset: str, parsed_snapshot_name: ParsedSnapshotName):
```

Comment:
Am I blind? 2nd argument is `dataset` but I don't see it getting used anywhere in this method?

Reply:
Oh I see, it's overwriting method based on class inheritance.